mpd/client.go

```diff
@@ -301,6 +301,11 @@ func (c *Client) SeekID(id, time int) error {
 	return c.okCmd("seekid %d %d", id, time)
 }
 
+//Seeks to the position TIME (in seconds; fractions allowed) within the current song.
+func (c *Client) SeekCur(time int) error {
```

Comment:
Seems like it would make for a nicer API to accept a `time.Duration` and then convert it to a float in seconds (if your comment is correct) using `d.Seconds()`, where `d` is a `time.Duration`.